src/Core/Models/DbConnectionParam.cs

```diff
@@ -10,10 +10,11 @@ namespace Azure.DataApiBuilder.Core.Models;
 /// </summary>
 public class DbConnectionParam
```

Comment:
Might not be in this PR but something to consider: Can we create a sub-classes of DbConnectionParam for MsSql Vs Other databases (like `MsSqlDbConnectionParam`) and create parameter objects for sub-classes? Seems like for MsSql, just SqlDbType should be sufficient. Please open up an issue if this makes sense.

Reply:
#2127